src/renderer/components/dock/dock.store.ts

```diff
@@ -282,6 +282,7 @@ export class DockStore implements DockStorageState {
 
         if (newTab?.kind === TabKind.TERMINAL) {
           // close the dock when selected sibling inactive terminal tab
+          // eslint-disable-next-line import/no-cycle
```

Comment:
ok because it's dynamically imported? No other workaround?

Reply:
No, it is fine because it is a dynamic import. 

In a previous iteration I had tried to refactor the docks, dock tabs, and terminals to not need this. But it was way too large of a change. I think I will come back at a later point to remove this.